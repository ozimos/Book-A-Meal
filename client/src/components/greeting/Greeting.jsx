import React from 'react';
import PropTypes from 'prop-types';
import { hot } from 'react-hot-loader';

const Greeting = props =>
  (
    <div>
      <h3>
                Welcome {props.isCaterer ? 'Caterer' : 'Customer'} {props.firstName}
      </h3>
    </div>
  );
Greeting.propTypes = {
  isCaterer: PropTypes.bool.isRequired,
  firstName: PropTypes.string.isRequired
};
export { Greeting };
export default hot(module)(Greeting);
