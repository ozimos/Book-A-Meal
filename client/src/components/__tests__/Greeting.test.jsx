/* global React:false, shallow:false */
/* eslint react/jsx-indent: off */

import Greeting from '../greeting/Greeting';

describe('Login Component', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Greeting isCaterer firstName="alias" />);
    expect(wrapper).toMatchSnapshot();
  });
});

