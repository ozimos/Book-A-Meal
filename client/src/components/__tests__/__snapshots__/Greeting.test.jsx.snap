// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Component renders correctly 1`] = `
<div>
  <h3>
    Welcome 
    Caterer
     
    alias
  </h3>
</div>
`;
